packages/replay/src/auth.ts

```diff
@@ -7,9 +7,20 @@ import path from "path";
 import { query } from "./graphql";
 import { getDirectory, maybeLog, openExecutable } from "./utils";
 import { Options } from "./types";
+import { getLaunchDarkly } from "./launchdarkly";
 
 const debug = dbg("replay:cli:auth");
 
+class GraphQLError extends Error {
+  constructor(message: string, public errors: any[]) {
+    const errorsMessage = errors
+      .map((e: any) => e.message)
+      .filter(Boolean)
+      .join(", ");
+    super(`${message}: ${errorsMessage}`);
+  }
+}
```

Comment:
Ideally, I want to throw directly in the `query` function but it's being used at a lot of places with conditional logic based on error. So, it should be a separate PR.